src/styles/font_names.js

```diff
@@ -0,0 +1,9 @@
+const fonts = {
+    // primary
+    space_mono: "Space Mono",
+    footerSizeDesktop: 20,
```

Comment:
i see you're still using `px` here - just make sure you have a backlog ticket to change these to ems when you start building out your type styles!